src/ui/qgsbookmarkeditordialog.ui

```diff
@@ -33,47 +54,49 @@
        </property>
       </widget>
      </item>
-     <item row="0" column="1">
-      <widget class="QLineEdit" name="mName"/>
-     </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblGroup">
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
+     <item row="0" column="1">
+      <widget class="QLineEdit" name="mName"/>
+     </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mGroup">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
-     <item row="3" column="0">
-      <widget class="QLabel" name="lblCRS">
-       <property name="text">
-        <string>CRS</string>
-       </property>
-      </widget>
-     </item>
-     <item row="3" column="1">
-      <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true">
+     <item row="2" column="0" colspan="2">
+      <widget class="QgsExtentGroupBox" name="mExtentGroupBox" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
+       <property name="title" stdset="0">
+        <string>Extent</string>
+       </property>
       </widget>
      </item>
-     <item row="4" column="0">
-      <widget class="QLabel" name="lblSaveLocation">
-       <property name="text">
-        <string>Saved in</string>
+     <item row="3" column="1">
+      <widget class="QDoubleSpinBox" name="mRotation">
```

Comment:
Please use a QgsDoubleSpinBox here, it allows for nice stuff like entering math equations in the widget itself (e.g. "5+10").

Reply:
Ok